example/src/App.js

```diff
@@ -3,15 +3,19 @@ import React, { Component } from "react";
 import ICalLink from "react-icalendar-link";
 
 export default class App extends Component {
-  render() {
+  render () {
     const style = { fontSize: "12px", padding: "5px" };
     const event = {
       title: "--- TITLE ---",
       description:
         ">>> DESCRIPTION <<<\\n Today is a huge day. \\n Tomorrow will be a huge day too!\\n",
       startTime: "2021-05-09T10:30:00+10:00",
       endTime: "2021-05-09T12:00:00+10:00",
-      location: "10 Carlotta St Artarmon NSW 2064"
+      location: "10 Carlotta St Artarmon NSW 2064",
+      attendees: [
+        "Joseph Chiang <josephj6802@gmail.com>",
+        "TC Wang <tingchun0113@gmail.com>"
```

Comment:
It seems your regular expression can handle more than these values. Can you provide them as well? 

btw, it's better to remove the real email since it's easy to cause spam.

```js
attendees: [
  "hello world<hello@world.com>",
  "<joo@foo.bar>"
]
```